src/components/patterns/horizontal-card.tsx

```diff
@@ -79,7 +81,7 @@ const HorizontalCard = ({
           )}
           <div>
             <div className="mb-16 flex flex-row flex-wrap items-center gap-16 @6xl:flex-nowrap">
-              <Image src="/card-rosette.png" alt="" className="object-contain" height={80} width={80} />
+              {!hideRosette && <RosetteIcon height={64} width={64} className="object-contain" />}
```

Comment:
FYI @pookmish I updated the conditional and name to match what we have in behaviors 😃